src/database/models/user.js

```diff
@@ -40,8 +45,10 @@ export default (sequelize, DataTypes) => {
   });
 
   User.prototype.generatePasswordHash = async function generatePasswordHash() {
-    const saltRounds = +process.env.SALT;
-    return bcrypt.hash(this.password, saltRounds);
+    if (!this.provider) {
+      const saltRounds = +process.env.SALT;
```

Comment:
You may need to revert your changes in all these files. These are all done and contributed by a different person